### PR TITLE
HPCC-30419 Add url-secret-name to ecl usage

### PR DESCRIPTION
**Pull request by jakesmith**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: ecl/eclcmd/eclcmd_shell.cpp [hunk 1]
```diff
@@ -205,24 +205,25 @@ void EclCMDShell::usage()
...
+           "   bundle           manage ECL bundles\n"
+           "   roxie            commands specific to roxie clusters\n"
+           "   abort            abort workunit(s) for WUID or job name\n"
+           "   status           show workunit(s) current status for WUID or job name\n"
+           "   getname          provide job name from WUID\n"
+           "   getwuid          provide WUID from job name\n"
+           "   zapgen           provide ZAP file from WUID\n"
+           "   sign             add digital signature to an ecl file or a text file\n"
+           "   listkeyuid       lists all the key user IDs that can be used in the sign command\n"
+           "   url-secret-name  Generate a secret name from a url for automatic url mapping"
```
Comment (afishbeck) [inline ecl/eclcmd/eclcmd_shell.cpp:226]: @jakesmith Looks great, but missing newline at end which gives us a blank line before the "Run" line below.

**Review by afishbeck — CHANGES_REQUESTED** [review]
Looks great just missing newline.

File: ecl/eclcmd/eclcmd_shell.cpp (hunk 1 above)
Comment (jakesmith) [reply ecl/eclcmd/eclcmd_shell.cpp]: @afishbeck - whoops yes, well spotted, now added (squashed in)

**Review by afishbeck — APPROVED** [review]
@jakesmith looks good.

**Pull request merged**